src/helpers/blockTime.js

```diff
@@ -0,0 +1,29 @@
+const MILLISECONDS_IN_A_SECOND = 1000
+
+export const blockTime = (eth) =>
+  async (blockNumber, showBlock = true, averageBlockTime = 13.965) => {
+    let timestamp
+    const currentBlock = await eth.getBlockNumber()
+
+    if (currentBlock >= blockNumber) {
+      timestamp = (await eth.getBlock(blockNumber)).timestamp * MILLISECONDS_IN_A_SECOND
```

Comment:
Note that when using some node endpoints that are served over a global cloud network (e.g. Infura), using `eth.getBlockNumber()` and `eth.getBlock()` together may not work as expected. You may get a higher latest block number from `eth.getBlockNumber()` than what is retrievable from `eth.getBlock()` (`eth.getBlock()` will return `{}`).

I would recommend just using a `eth.getBlock('latest')` above and going off of that. The `'latest'` is accepted in the JSONRPC spec and should guarantee that we at least get a close-to-latest block from the network.